liquibase-standard/src/main/java/liquibase/snapshot/jvm/IndexSnapshotGenerator.java

```diff
@@ -83,9 +83,12 @@ protected void addTo(DatabaseObject foundObject, DatabaseSnapshot snapshot) thro
                     }
                     Boolean descending = "D".equals(ascOrDesc) ? Boolean.TRUE : ("A".equals(ascOrDesc) ? Boolean
                             .FALSE : null);
-                    index.addColumn(new Column(row.getString("COLUMN_NAME")).setComputed(false).setDescending(descending).setRelation(index.getRelation()));
+                    Boolean included = row.getBoolean("IS_INCLUDED_COLUMN");
```

Comment:
Where is it grabbing this value from?

Reply:
the jdbc connections metadata:
https://javadoc.io/doc/com.h2database/h2/1.4.198/org/h2/jdbc/JdbcConnection.html#getMetaData()

Reply:
Sorry, I meant where is the property `IS_INCLUDED_COLUMN` defined? Shouldn't it be just `INCLUDED`?

Reply:
I believe it's value is defined in the external JDBC Library. I originally found its value by watching the values liquibase is returning in this method.